.github/workflows/pylint_sonarqube.yml

```
@@ -0,0 +1,63 @@
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10"]
+    
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0  # Necessário para SonarCloud
+    
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)